transform/mattermost-analytics/models/intermediate/web_app/platform/int_mm_telemetry_prod_performance_events.sql

```diff
@@ -11,12 +11,8 @@ SELECT
 FROM
     {{ ref('stg_mm_telemetry_prod__performance_events') }}
 WHERE
-    id not in (
-        'd\';waitfor/**/delay\'0:0:0\'/**/--/**/',
-        's\');waitfor/**/delay\'0:0:0\'/**/--/**/',
-        'v\';waitfor/**/delay\'0:0:2\'/**/--/**/',
-        'o\');waitfor/**/delay\'0:0:0\'/**/--/**/'
-    )
+    -- Exclude non-UUID strings
+    not id ilike '%waitfor%'
```

Comment:
there could be more ids that are having some invalid strings in them other than waitfor - wondering if we should open a ticket to expand this check to make it more general (and not only for perf events).

Reply:
We can replace it with a regex to check if id is a UUID. But this could break if rudderstack decides to change the format of id. 

Given the frequency of these issues, I think this change should be enough for now.